### ISSUE TITLE
Bug in School Administrator Dashboard

### ISSUE THREAD

**Issue by alexsoble** [studentinsights/studentinsights#1341]
@edavidsonsawyer, I logged into the production app and navigated to the School Administrator Dashboard using the test user account. The page didn't render and I got this error in the console: 

![screen shot 2017-12-23 at 9 46 38 am](https://user-images.githubusercontent.com/3209501/34321977-67207c8c-e7d0-11e7-8799-5fdc982261d2.png)

The test user account may have created some different `current_educator` properties that the dashboard hasn't handled yet. Or it could be an unrelated error. 

Either way, this isn't urgent since this is a "shipped-dark" feature and it's the holidays/weekend. But wanted to put it on your radar @edavidsonsawyer for when we're back in high-usage mode!

**Comment by edavidsonsawyer** [comment]
Hey @alexsoble, I didn't realize it when I opened the PR, but I think this is fixed by #1350. Just needed to switch up the dashboard bar chart to import HighChartsWrapper instead of using window.shared.

I think it would be a good idea if  HighchartsWrapper.js was moved into /helpers since dashboards and anything else that wants to use a chart will likely need it, and its current location doesn't suggest any dependencies. If that's alright, I'll just quickly do a PR with that. Meanwhile, this is fixed once #1350 is in.
